### ISSUE TITLE
DNS Server Configuration

### ISSUE THREAD

**Issue by atulnarang** [bradcbennett/PrimeSuper.com.au#42]
Brad i have been researching on DNS configuration for windows Azure server, although its possible to create your own name server but needs some network expert, we would need some time to research and implement.

i would advise use of "A" records but as we discussed that's not possible, one more recommendation would be shifting of domain to alternate provider like crazydomains or name.com

**Comment by atulnarang** [comment]
you can get more insight at this link:
http://www.microsoftnow.com/2013/10/hosting-your-public-dns-servers-on-windows-azure.html

**Comment by bradcbennett** [comment]
I think I've sorted it. Would be good to have you check that everything looks OK.

I've set up a Cloudflare account. At Cloudflare I set the DNS to point to the current site. I then delegated the domain to use Cloudflare's DNS via Domain Registration Services. 
As we have control of the DNS through Cloudflare we can make the IP change on Monday morning. Screenshot below of the interface.

Does this set up make sense to you?

![image](https://cloud.githubusercontent.com/assets/4900157/10088808/ea4fcf5a-6360-11e5-854c-ef431708ca31.png)
